src/routes/index.ts

```diff
@@ -1,6 +1,7 @@
 enum Routes {
   'ApiDocumentation' = 'api-documentation',
   'ArticleList' = 'article-list',
+  'PostArticle' = 'post-article',
```

Comment:
Suggest we call it `CreateArticle`. Maybe even `AddArticle`?

Reply:
Will go for `AddArticle` as it matches the method call.